.github/workflows/build.yml

```diff
@@ -46,3 +46,25 @@ jobs:
       uses: AndreMiras/coveralls-python-action@develop
       with:
         parallel-finished: true
+
+  build_doc:
```

Comment:
Is this needed? [RTD can do it for us](https://docs.readthedocs.io/en/stable/pull-requests.html), and they provide extensive logs for debugging.

Reply:
Yea that is probably the best thing to do.  I was trying to get around using RTD for now just so that people could see that.